build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -266,10 +266,9 @@ flowExporter:
   # <HOST> to <Service namespace>/<Service name>. For example,
   # "flow-aggregator/flow-aggregator" can be provided to connect to the Antrea
   # Flow Aggregator Service.
-  # If PORT is empty, we default to 4739, the standard IPFIX port.
-  # If no PROTO is given, we consider "tls" as default. We support "tls", "tcp" and
-  # "udp" protocols. "tls" is used for securing communication between flow exporter and
-  # flow aggregator.
+  # If PORT is empty, we default to 14739. If no PROTO is given, we consider "grpc" as default.
+  # We support "grpc", "tls", "tcp" and "udp" protocols. "grpc", "tls" are used for securing
+  # communication between flow exporter and flow aggregator.
```

Comment:
This doesn't seem to match the code: https://github.com/antrea-io/antrea/blob/8bfa664e7ca85062236bb79fa08bea60b923ca2c/cmd/antrea-agent/options.go#L51C2-L53

While the *chart* defaults to `"flow-aggregator/flow-aggregator:14739:grpc"`, it looks like the code doesn't use the same default (if using a custom config where this address, or part of it, is omitted).

I can't recall if there is a reason for the difference in behavior... I think that maybe the intent was that if a user has a custom config, we should *not* change the behavior in an unexpected way. Hence why the code was not updated.

Reply:
Ah, ok. I thought the chart value is the default one. Do we need to update the protocols?